index.bs

```diff
@@ -7997,9 +8015,18 @@ defining which global environments the construct can be used in.
 The [=exposure set=]
 for a given construct is defined as follows:
 
-*   If the [{{Exposed}}]
-    [=extended attribute=]
-    is specified on the construct, then the [=exposure set=]
+*   If the [{{Exposed}}] [=extended attribute=] [=takes a wildcard=], then the
```

Comment:
So unfortunately this bit has seen a lot of changes since you opened the initial bug report. I _think_ you should just be able to describe the **own exposure set** of constructs declared with [Exposed=*] as containing all of the interfaces which have a [{{Global}}] [=extended attribute=] and be done with it.

Reply:
Not sure how I forgot to rebase before putting this patch up. Fixed. What do you think of the new wording?